python/cuproj/pyproject.toml

```diff
@@ -26,7 +26,7 @@ requires = [
 
 [project]
 name = "cuproj"
-version = "23.12.0"
+version = "23.12.00"
```

Comment:
Aligning with other RAPIDS projects -- these lines are supposed to be "full versions" and not "PEP 440 versions." I'm not sure how this typo occurred, because the script is set to do this correctly as far as I can tell.